packages/apps/job-launcher/server/src/common/constants/index.ts

```diff
@@ -0,0 +1,17 @@
+export const NS = 'hmt';
+export const COMPANY_NAME = 'Human Protocol Foundation';
+export const COINGECKO_API_URL =
+  'https://api.coingecko.com/api/v3/simple/price';
+export const JOB_RETRIES_COUNT_THRESHOLD = 3;
+export const TX_CONFIRMATION_TRESHOLD = 1;
+
+export const EXCHANGE_ORACLE_ADDRESS = '';
+export const EXCHANGE_ORACLE_WEBHOOK_URL =
+  'http://localhost:3005/webhook/job-launcher';
+export const RECORDING_ORACLE_ADDRESS = '';
+export const REPUTATION_ORACLE_ADDRESS = '';
+export const JOB_LAUNCHER_FEE = 50;
+export const RECORDING_ORACLE_FEE = 50;
+export const REPUTATION_ORACLE_FEE = 50;
+
+export const S3_PORT = 9000;
```

Comment:
Why are those values constants? Those should be obtained from env variables

Reply:
In our last conversation, we discussed with you that we transfer these variables to constants.